pubspec.yaml

```diff
@@ -21,4 +21,4 @@ dev_dependencies:
   args: ">=1.0.0 <2.0.0"
   http: ">0.11.0 <0.12.0"
   path: "^1.5.1"
-  test: ">=0.12.18+1 <0.13.0"
+  test: ">=0.12.18+1 <2.0.0"
```

Comment:
Any reason no to set test to ^1.3.0 or something like that?

Reply:
Still supporting Dart 1.x